src/components/Auth/Logout.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { useKeycloak } from '@react-keycloak/web';
+import { Redirect } from 'react-router';
+
+export const Logout = () => {
+  const { keycloak, initialized } = useKeycloak();
+
+  const logout = ( keycloak ) => {
+    if (keycloak.authenticated){
```

Comment:
this feels a bit weird. I think you can describe how this component is implemented all in the return.